dp/cloud/go/services/dp/storage/db/table.go

```diff
@@ -43,6 +43,10 @@ func addColumns(builder sqorc.CreateTableBuilder, fields FieldMap) sqorc.CreateT
 			colBuilder = colBuilder.Default(field.DefaultValue)
 		}
 		builder = colBuilder.EndColumn()
+
+		if field.Unique {
```

Comment:
Does this work when there are multiple unique columns? Maybe add one simple test to `query_test.go` for it

Reply:
done